tests/app/Infrastructure/Controller/GetsWalletBalanceControllerTest.php

```diff
@@ -24,8 +24,8 @@ public function throwsErrorWhenWalletIdNotFound()
 
         $response = $this->get('api/wallet/' . $walletOne->getWalletId() . '/balance');
 
-        $response->assertNotFound();
-        $response->assertExactJson(['description' => 'A wallet with the specified ID was not found']);
+        $response->assertBadRequest();
```

Comment:
```suggestion
                $response->assertStatus(Response::HTTP_BAD_REQUEST);
```
Change this in all the tests